src/patterns/components/highlight-board/no-image.hbs

```diff
@@ -0,0 +1,6 @@
+---
+order: 2
+description: This variation of the highlight board component should be used when there is an alternative to a
```

Comment:
We decided to capitalize all component names,  highlight board should be  Highlight Board